Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Authorization.Application.DependencyInjection;
+using Duende.IdentityServer.EntityFramework.DbContexts;
+using Duende.IdentityServer;
+using Microsoft.EntityFrameworkCore;
+using Duende.IdentityServer.Models;
+using Duende.IdentityServer.EntityFramework.Mappers;
+using IdentityModel;
+
+namespace Authorization.Presentation.DependencyInjection
+{
+    public static class PresentationDependencyInjection
+    {
+        public static IServiceCollection AddPresentationDependencyInjection(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddProblemDetails();
+
+            services.AddApplicationDependencyInjection(configuration);
+
+            services.AddRazorPages();
+
+            var connectionString = configuration.GetConnectionString("DefaultConnection");
+            var isBuilder = services
```

Comment:
What is isBuilder for?

Reply:
It was autogenerated, it is useless, removed